avocado/plugins/human.py

```diff
@@ -61,6 +61,11 @@ def start_test(self, result, state):
             name = "<unknown>"
             uid = '?'
         if self.runner == 'nrunner':
+            # ignore tasks that are not tests
+            task_category = state.get('task_category')
+            if task_category is not None and task_category != 'test':
```

Comment:
I don't think this is the right place for this. IIUIC the `ResultEvents` interface is for the test events and IMO it doesn't make sense to trigger  `start_test` and `end_test` events when the task is not a test. Maybe we should consider updating the `plugin_interfaces` such as we will have events triggered by a task and events triggered by a special type of task like `test`.

Reply:
I agree with you this change is specific. As we still don't have a clear vision of where the `Task` architecture is evolving, I thought it would be better to do a small and specific change than changing the whole `pluging_interface`. One of the reasons is that, soon, we may have a `Task` that needs to execute something related to the `ResultEvents`. 

In a nutshell, the idea here was: introduce the task category for flexibility and future extension; and make the smallest change needed to allow the addition of the Requirements Resolver architecture without breaking the human UI, leaving bigger and complex changes to the near future.

Let me know if it makes sense.

Reply:
I agree with @richtja here. Also, other plugins can be reading this list, and it will show non-tests there too. So if we would like to filter those tests, this should be done internally.

Reply:
Okay, got it! I'll send a new version with some changes.